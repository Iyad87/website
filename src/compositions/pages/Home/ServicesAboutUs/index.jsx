import React from 'react';
import TileGrid from '../../../../components/TileGrid';

const ServicesAboutUs = () => null;

export default ServicesAboutUs;
