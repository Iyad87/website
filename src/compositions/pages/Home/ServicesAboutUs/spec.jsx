import React from 'react';
import { shallow } from 'enzyme';
import ServicesAboutUs from './index';

test('compositions/homepage/ServicesAboutUs snapshot', () => {
  const wrapper = shallow(<ServicesAboutUs />);

  expect(wrapper).toMatchSnapshot();
});
