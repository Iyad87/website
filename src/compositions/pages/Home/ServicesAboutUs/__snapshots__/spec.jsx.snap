// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/homepage/ServicesAboutUs snapshot 1`] = `
<styled.section
  wide={true}
>
  <styled.ul>
    <styled.li>
      <styled.div>
        <styled.h5>
          Services
        </styled.h5>
        <styled.p>
          We enable your organisation to deliver world-class technology and user experiences, long after we leave.
        </styled.p>
      </styled.div>
    </styled.li>
    <styled.li
      colour="darker"
    >
      <styled.div>
        <styled.h5>
          About Us
        </styled.h5>
        <styled.p>
          We have team members from a broad spectrum of backgrounds and incredible technology companies.
        </styled.p>
      </styled.div>
    </styled.li>
  </styled.ul>
</styled.section>
`;
